packages/moonstone/styles/colors.less

```diff
@@ -113,6 +113,13 @@
 @moon-formcheckbox-bg-color: fade(@moon-white, 22%);
 @moon-formcheckbox-text-color: @moon-white;
 @moon-formcheckbox-border-color: fade(@moon-white, 15%);
+@moon-formcheckbox-focus-bg-color: @moon-spotlight-color;
+@moon-formcheckbox-focus-text-color: @moon-spotlight-text-color;
+
+// FormCheckboxItem
+// ---------------------------------------
+@moon-formcheckboxitem-focus-text-color: @moon-spotlight-text-color;
```

Comment:
:yuno: have this one alphabetized?

Reply:
Text usually comes before background color. These are usually grouped by state rather than alphabetically. Very few lines are alphabetically sorted in this file.